package.json

```diff
@@ -67,22 +67,27 @@
     "@types/xml2js": "0.0.32",
     "@types/xmldom": "^0.1.28",
     "atob": "^2.0.3",
+    "bufferutil": "^3.0.0",
     "es6-shim": "^0.35.2",
     "loglevel": "^1.4.1",
     "pako": "^1.0.3",
     "rxjs": "^5.3.0",
     "typescript-collections": "^1.2.3",
+    "utf-8-validate": "^3.0.1",
     "uuid": "^3.0.1",
+    "ws": "^3.0.0",
```

Comment:
because signalr needs much more work to adapt in our api ? or it's not working with nodejs ?